plugin/src/test/kotlin/org/openrewrite/gradle/RewriteDryRunTest.kt

```diff
@@ -122,4 +122,76 @@ class RewriteDryRunTest : RewritePluginTest {
         assertThat(rewriteDryRunResult.outcome).isEqualTo(TaskOutcome.SUCCESS)
         assertThat(File(projectDir, "build/reports/rewrite/rewrite.patch").exists()).isTrue
     }
+
+    @Test
+    fun testMultiplatform(@TempDir projectDir: File) {
+        gradleProject(projectDir) {
+            buildGradle("""
+                plugins {
+                    id("java")
+                    id("org.openrewrite.rewrite")
+                    id("org.jetbrains.kotlin.multiplatform") version "1.8.0"
+                }
+                group = "org.example"
+                version = "1.0-SNAPSHOT"
+                
+                repositories {
+                    mavenLocal()
+                    mavenCentral()
+                    maven {
+                       url = uri("https://oss.sonatype.org/content/repositories/snapshots")
+                    }
+                }
+
+                kotlin {
+                    jvm {
+                        jvmToolchain(8)
+                        withJava()
+                    }
+
+                    sourceSets {
+                        commonMain {
+                            dependencies {
+                            }
+                        }
+                        commonTest {
+                            dependencies {
+                            }
+                        }
+                        jvmMain {
+                        }
+                        jvmTest {
+                        }
+                    }
+                }
+            """)
+            settingsGradle("""
+                pluginManagement {
+                    repositories {
+                        mavenLocal()
+                        maven { url = uri("https://plugins.gradle.org/m2") }
+                        gradlePluginPortal()
+                    }
+                }
+                rootProject.name = "example"
+            """)
+            sourceSet("commonMain") {
+                kotlin("""
+                    class HelloWorld {
+                        fun sayHello() {
+                            println("Hello world")
+                        }
+                    }
+                """)
+            }
+        }
+        val gradleVersion = System.getProperty("org.openrewrite.test.gradleVersion")
+        if (gradleVersion >= "6.1") {
```

Comment:
On older versions of Gradle this test does nothing at all, except waste time writing files to a temporary folder which is immediately deleted. Use an appropriate annotation to disable it completely on versions of gradle where this is not expected to work

Reply:
Resolved by: https://github.com/openrewrite/rewrite-gradle-plugin/pull/180/commits/82deed993bd368055a49d6f50d59e3d752fafcef